tools/api-tests/yarn.lock

```diff
@@ -4197,7 +4199,7 @@ lodash.isstring@^4.0.1:
   resolved "https://registry.yarnpkg.com/lodash.isstring/-/lodash.isstring-4.0.1.tgz#d527dfb5456eca7cc9bb95d5daeaf88ba54a5451"
   integrity sha1-1SfftUVuynzJu5XV2ur4i6VKVFE=
 
-lodash.merge@^4.6.0, lodash.merge@^4.6.1, lodash.merge@^5.6.2:
```

Comment:
I remember this causing problems before because there is no `5.x` version of `lodash`

Reply:
Pipeline looks ok - so let's trust it?

Reply:
I agree with @diversemix , trust the pipeline, if it breaks again, ensure the tests pick it up next time. test test test